pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -528,9 +534,17 @@ def handle_command(
         scipy_fixes(new_args)
 
     returncode = subprocess.run(new_args).returncode
+
     if returncode != 0:
         sys.exit(returncode)
 
+    # Rust gives output files a `.wasm` suffix, but we need them to have a `.so`
+    # suffix.
+    if line[0:2] == ["cargo", "rustc"]:
+        p = Path(args.builddir)
+        for x in p.glob("**/*.wasm"):
+            shutil.copy(x, x.with_suffix(".so"))
```

Comment:
Copy or rename? Is there a reason to keep the original ones? Also so they are not included in the final wheel.

Reply:
Yeah clearly move is the right approach.